src/main/kotlin/no/nav/syfo/cronjob/dialogmotesvar/DialogmotesvarProducer.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.syfo.cronjob.dialogmotesvar
+
+import no.nav.syfo.dialogmote.api.domain.KDialogmotesvar
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class DialogmotesvarProducer(
+    private val kafkaDialogmotesvarProducer: KafkaProducer<String, KDialogmotesvar>,
+) {
+    fun sendDialogmotesvar(
+        kDialogmotesvar: KDialogmotesvar,
```

Comment:
Er `k`-prefixen noe kafka-format som er vanlig å bruke?

Reply:
Ja, i dette tilfellet blir en en dto for det man legger på topic. Bruker vanlig jackson-json-serialisering (som i api'ene).

Går også an å bruke avro-skjema dersom man vil publisere meldingsformatet på en mer strukturert måte.